.github/workflows/accessibility.yml

```diff
@@ -0,0 +1,5 @@
+- name: A11yWatch - The Accessibility Suite
+  uses: a11ywatch/github-action@v1.2.44
+  with:
+    WEBSITE_URL: ${{ secrets.https://docs.suborbital.dev }}
```

Comment:
This does not look right :thinking:

Reply:
> This does not look right 🤔

🤷‍♀️ I just followed this: https://github.com/marketplace/actions/a11ywatch-the-accessibility-suite